src/main/java/org/folio/dew/batch/bulkedit/jobs/BulkEditInstanceProcessor.java

```diff
@@ -0,0 +1,120 @@
+      .modeOfIssuanceId(instanceReferenceService.getModeOfIssuanceNameById(instance.getModeOfIssuanceId(), errorServiceArgs))
+      .administrativeNotes(isEmpty(instance.getAdministrativeNotes()) ? EMPTY : String.join(ITEM_DELIMITER, escaper.escape(instance.getAdministrativeNotes())))
+      .title(instance.getTitle())
+      .indexTitle(instance.getIndexTitle())
+      .series(fetchSeries(instance.getSeries()))
+      .contributors(fetchContributorNames(instance.getContributors()))
+      .editions(isEmpty(instance.getEditions()) ? EMPTY : String.join(ITEM_DELIMITER, escaper.escape(new ArrayList<>(instance.getEditions()))))
+      .physicalDescriptions(isEmpty(instance.getPhysicalDescriptions()) ? EMPTY : String.join(ITEM_DELIMITER, escaper.escape(instance.getPhysicalDescriptions())))
+      .instanceTypeId(instanceReferenceService.getInstanceTypeNameById(instance.getInstanceTypeId(), errorServiceArgs))
+      .natureOfContentTermIds(fetchNatureOfContentTerms(instance.getNatureOfContentTermIds(), errorServiceArgs))
+      .instanceFormatIds(fetchInstanceFormats(instance.getInstanceFormats(), errorServiceArgs))
+      .languages(isEmpty(instance.getLanguages()) ? EMPTY : String.join(ITEM_DELIMITER, escaper.escape(instance.getLanguages())))
+      .publicationFrequency(isEmpty(instance.getPublicationFrequency()) ? EMPTY : String.join(ITEM_DELIMITER, escaper.escape(new ArrayList<>(instance.getPublicationFrequency()))))
+      .publicationRange(isEmpty(instance.getPublicationRange()) ? EMPTY : String.join(ITEM_DELIMITER, escaper.escape(new ArrayList<>(instance.getPublicationRange()))))
+      .build();
+
+
+    return instanceFormat.withOriginal(instance);
+  }
+
+  private String fetchInstanceFormats(List<FormatOfInstance> instanceFormats, ErrorServiceArgs errorServiceArgs) {
+    return isEmpty(instanceFormats) ? EMPTY :
+      instanceFormats.stream()
+        .map(iFormat -> instanceReferenceService.getFormatOfInstanceNameById(iFormat.getId(), errorServiceArgs))
+        .map(iFormatName -> String.join(ITEM_DELIMITER, escaper.escape(iFormatName)))
+        .collect(Collectors.joining(ITEM_DELIMITER));
+  }
+
+  private String fetchNatureOfContentTerms(Set<String> natureOfContentTermIds, ErrorServiceArgs errorServiceArgs) {
+    return isEmpty(natureOfContentTermIds) ? EMPTY :
+      natureOfContentTermIds.stream()
+        .map(natId -> instanceReferenceService.getNatureOfContentTermNameById(natId, errorServiceArgs))
+        .map(natName -> String.join(ITEM_DELIMITER, escaper.escape(natName)))
+        .collect(Collectors.joining(ITEM_DELIMITER));
+  }
+
+  private String fetchContributorNames(List<InstanceContributorsInner> contributors) {
+    return isEmpty(contributors) ? EMPTY :
+      contributors.stream()
+        .map(c -> String.join(ITEM_DELIMITER, escaper.escape(c.getName())))
+        .collect(Collectors.joining(ITEM_DELIMITER));
```

Comment:
Contributor names should be delimited with semicolon and spaces (`" ; "`)

Reply:
@siarhei-charniak 
done